src/test/java/edu/ucsb/ucsbcslas/controllers/RoomSlotControllerTests.java

```diff
@@ -0,0 +1,193 @@
+package edu.ucsb.ucsbcslas.controllers;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import java.lang.IllegalArgumentException;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import static org.junit.Assert.fail;
+
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.csrf;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.ucsb.ucsbcslas.advice.AuthControllerAdvice;
+import edu.ucsb.ucsbcslas.entities.RoomSlot;
+import edu.ucsb.ucsbcslas.repositories.RoomSlotRepository;
+import edu.ucsb.ucsbcslas.entities.ActiveQuarter;
+import edu.ucsb.ucsbcslas.repositories.ActiveQuarterRepository;
+import edu.ucsb.ucsbcslas.models.Course;
+import jdk.jfr.Timestamp;
```

Comment:
This import freaked me out because I thought you were using an internal JDK class. I see that this isn't actually being used, can you clean up this import (and any other unused imports)?